application/ai/main_plot_llm_contract.py

```
@@ -0,0 +1,103 @@
+# ---------------------------------------------------------------------------
+
+
+class LlmPlotOption(BaseModel):
+    """单条主线情节选项。"""
+
+    model_config = ConfigDict(extra="forbid")
+
+    id: str = Field(min_length=1, max_length=64, description="选项唯一标识")
+    type: str = Field(min_length=1, max_length=64, description="情节类型")
+    title: str = Field(
+        min_length=1, max_length=16, description="情节标题（8-16 字）",
```

Ruff (0.15.10)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)